PmlUnit/CodeEditor.cs

```diff
@@ -274,6 +277,22 @@ protected override IEnumerable<string> GetExtraArguments(string fileName, int li
         }
     }
 
+    class PMLStudioCodeEditor : BaseCodeEditor
```

Comment:
rename to `PmlStudioCodeEditor`
```suggestion
    class PmlStudioCodeEditor : BaseCodeEditor
```